.github/CODEOWNERS

```diff
@@ -94,7 +94,7 @@
 
 
 # ==============================================================
-#  binary-com/deriv-shared
+#  binary-com/@deriv/shared
```

Comment:
Please revert all changes from this file. None of these are actual Github teams.